Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocControlObject.cs

```diff
@@ -69,39 +69,40 @@ protected IReadOnlyList<string> GetValidationErrors ([NotNull] ElementScope scop
       if (IsValid (scope))
         return new List<string>();
 
-      ElementScope validationErrorsScope;
       string describedBy;
 
       try
       {
-        describedBy = scope["aria-describedby"];
+        describedBy = scope.GetAttributeOrThrow ("aria-describedby");
       }
       catch (MissingHtmlException exception)
       {
-        throw new MissingHtmlException (
-            "Could not find validation error field. This could be due to wrong markup or a missing validators.",
-            exception);
+        throw GetMissingValidationErrorFieldException (exception);
       }
 
       var validationErrorIDs = describedBy.Split (new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
 
+      var validationErrorIndex = int.Parse (scope.GetAttributeOrThrow (DiagnosticMetadataAttributes.ValidationErrorIDIndex));
+      var validationErrorsScope = Scope.FindId (validationErrorIDs[validationErrorIndex]);
+
+      // re-motion renders the delimiter as <br />, but the browser (and Selenium) show it as <br>
+      const string validationErrorDelimiter = "<br>";

```

Comment:
Do we have a browser-based test for this?

Reply:
`ValidationErrorTestCaseFactory` defines test cases using the `BocControlObject.GetValidationErrors()` method. A change of the delimiter string leads to multiple failing tests.